src/components/chats/ChatItem/ChatItem.tsx

```diff
@@ -155,7 +157,11 @@ export default function ChatItem({
       )}
     >
       {!isMyMessage && (
-        <AddressAvatar address={ownerId} className='flex-shrink-0' />
+        <AddressAvatar
+          address={ownerId}
+          avatar={authUser!.profilePic}
```

Comment:
this one is a problem, because the condition here is !isMyMessage, so if its not your message, why put the current authUser.profilePic?

and it errors out if you haven't logged in because authUser is undefined

Reply:
That makes sense. That's why the profile pictures are all looking the same.